pyasdf/tags/transform/projections.py

```diff
@@ -69,6 +69,74 @@ def assert_equal(cls, a, b):
         assert_array_equal(a.angle, b.angle)
 
 
+class Rotate3DType(TransformType):
+    name = "transform/rotate3d"
```

Comment:
All three of these types have the same YAML tag name, so if the parser encounters that tag, it doesn't know which of the three types to create (it will probably just make the last defined one below).  What's the difference between `RotateNative2Celestial` and `RotateCelestial2Native` anyway?

If we need a separate type to distinguish between the direction, we'll need another YAML tag -- or use the same YAML tag but another key on the object to specify the direction.  (That will require updating the schema, which I just merged without reading this -- sorry).